book/src/intro.md

```diff
@@ -0,0 +1,33 @@
+# Introduction
+
+Taiga is a shielded state transition protocol that allows applications built on top of it to enjoy the advantages of fully shielded multi-party state transitions 
+(hiding the application type, the data associated with it, involved parties, etc) without giving up the application complexity.
+
+![img.png](images/Intro_Taiga_Apps.png)
```

Comment:
Would it make sense to have the Apps boxes inside the Taiga box? I see Taiga as the underlying protocol behind every app

Reply:
I see it the same way and I agree that arrows aren't good, but I'm not sure about the boxes inside one box either. Kinda gives a feeling that the amount of boxes that the Taiga box can fit in is limited. When I think about Taiga applications, the diagrams for ISO/OSI model come to mind, but it is hard to copy it with only two layers. I'll see what I can do, have a couple of ideas